### PR TITLE
Added several features

### PR DESCRIPTION
**Pull request by noob339**
Added camera feature, added labels feature, fixed navigation issues, made custom button components

### CODE REVIEW

**Comment by qingquan-li** [comment]
Great, i'll review it tomorrow.

File: App.tsx [hunk 1]
```diff
@@ -1,9 +1,16 @@
 import * as React from 'react';
 
 import Navigation from './navigation';
+import Knob from './screens/KnobLabel';   //for testing before implementing with navigation 
+import Door from './screens/DoorLabel';  //^
+import Misc from './screens/Misc';      //^
```
Comment (qingquan-li) [inline App.tsx]: What does "/^" mean? It'll be better if you can add some comments here.

File: components/customPostButton.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        >
+            <Text style={styles.buttonText}>{title}</Text>
+
+        </TouchableOpacity>
+    );
+  };
+  
+  export default CustomButton;
+  
+  const styles = StyleSheet.create({
```
Comment (qingquan-li) [inline components/customPostButton.tsx]: Either 4 spaces or 2 spaces for indentation is fine, but don't mix them.
By convention, we use 2 spaces for indentation in DoorGo codebase.

File: navigation/index.tsx [hunk 3]
```diff
@@ -39,14 +44,58 @@ function ProfileStackScreen() {
...
+const CamNav = createNativeStackNavigator();
+
+function CameraStackNav()
+{
+  return (
+    <CamNav.Navigator
+      
+    >
+
+    <CamNav.Screen
```
Comment (qingquan-li) [inline navigation/index.tsx]: Please format the indentation. You can google how to format the code.
Like, on VSC, you can use shift+option(alt)+f to format the code.

File: screens/Misc.tsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
+import Checkbox from 'expo-checkbox';
+import React, { useState } from 'react';
+import { StyleSheet, Text, View, Pressable} from 'react-native';
+import CustomButton from '../components/customPostButton';
+
+
+export default function Misc({navigation}: {navigation: any})
```
Comment (qingquan-li) [inline screens/Misc.tsx]: You can add more comments to let other cooperators know what the code is doing. Without any comments in a code file or function, in the future, you may also forget how these codes work.

Add comments to functions to let developers know what the function does. Ex:
```
/**
 * Adds two numbers.
 * @param {number} num1 The first number to add.
 * @param {number} num2 The second number to add.
 * @return {number} The result of adding num1 and num2.
 */
function bigThing() {
    // ...
}
```

**Review by qingquan-li — CHANGES_REQUESTED** [review]
@noob339 You did an excellent job. I reviewed your code and added some comments. Mainly about the code format and code comments.
It'll be great if you can add more code comments to let other developers know how your code (functions) work.

BTW, when I run the code, there was a console warning: "Found screens with the same name nested inside one another, Check: Post, Post > Post." Can you fix it?

File: App.tsx (hunk 1 above)
Comment (noob339) [reply App.tsx]: it means it means that the comment above it applies to it as well, sorry will make more comments moving forward

File: components/customPostButton.tsx (hunk 2 above)
Comment (noob339) [reply components/customPostButton.tsx]: Got it, do you use a tool to auto format at the end?

File: navigation/index.tsx (hunk 3 above)
Comment (noob339) [reply navigation/index.tsx]: Sounds good, again my apologies

File: screens/Misc.tsx (hunk 4 above)
Comment (noob339) [reply screens/Misc.tsx]: Yes, I will leave more comments

File: components/customPostButton.tsx (hunk 2 above)
Comment (qingquan-li) [reply components/customPostButton.tsx]: i don't use tools. Absolutly, you can use a tool to auto format js/ts code.

Comment (noob339) [reply components/customPostButton.tsx]: Used prettier formatter to format code

File: screens/Misc.tsx (hunk 4 above)
Comment (noob339) [reply screens/Misc.tsx]: Need to add more comments but left some for now, will work on locations and add comments as I work

**Review by qingquan-li — APPROVED** [review]

**Pull request merged**